Assets/Scripts/NursePatrol.cs

```diff
@@ -67,8 +79,6 @@ void GotoNextPoint()
     // Follows Target if they are in the field of vision of the nurse, called in FieldOfVision.cs
```

Comment:
This is not being called in FieldOfVision anymore right?

Reply:
Yea instead we call it in `NursePatrol`, and `FieldOfVision` instead calls `HandleTargetsInRange`.

Reply:
Ok yeah, the comment just confused me for a sec but I figured it out after!